mteb/tasks/PairClassification/ind/IndoNLI.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskPairClassification import AbsTaskPairClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class IndoNLI(AbsTaskPairClassification):
+    metadata = TaskMetadata(
+        name="indonli",
+        dataset={
+            "path": "indonli",
+            "revision": "3c976110fc13596004dc36279fc4c453ff2c18aa",
+        },
+        description="IndoNLI is the first human-elicited Natural Language Inference (NLI) dataset for Indonesian. IndoNLI is annotated by both crowd workers and experts.",
+        reference="https://link.springer.com/chapter/10.1007/978-3-030-41505-1_39",
+        type="PairClassification",
+        category="s2s",
+        eval_splits=["test_expert"],
+        eval_langs=["ind-Latn"],
+        main_score="ap",
+        date=("2021-01-01", "2021-11-01"),  # best guess
+        form=["written"],
+        domains=["Encyclopaedic", "Web", "News"],
+        task_subtypes=["Textual Entailment"],
+        license="CC-BY-SA 4.0",
+        socioeconomic_status="mixed",
+        annotations_creators="human-annotated",
```

Comment:
Shouldn't it be `expert-annotated` according to the description and since we're only using the split that was annotated by experts

Reply:
Nevermind it was corrected